src/pipeline_v3/legacy_backup/llama_index_files/test_llamaindex_postgresql.py

```
@@ -0,0 +1,415 @@
+        return config
+
+    @pytest.fixture
+    def sqlite_config(self):
+        """Create a SQLite configuration."""
+        config = PipelineConfig()
+        config.database.backend = "sqlite"
+        return config
+
+    def test_index_manager_with_postgresql_factory(self, pg_config):
+        """Test IndexManager initialization with PostgreSQL via DatabaseFactory."""
+        with patch('storage.database_factory.DatabaseFactory') as mock_factory_class:
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)